### PR TITLE
Run3-gex56 Try to add test script for overlap of dd4hep geometry

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Try to add test script for overlap of dd4hep geometry

#### PR validation:

Add eras in the script - but the the code needs debugging

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

File: SimG4Core/PrintGeomInfo/test/python/g4OverlapCheck_dd4hep_cfg.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# Geant4 geometry check 
+process.g4SimHits.G4CheckOverlap.OutputBaseName = cms.string("cms2021")
+process.g4SimHits.G4CheckOverlap.OverlapFlag = cms.bool(True)
+process.g4SimHits.G4CheckOverlap.Tolerance  = cms.double(0.01)
+process.g4SimHits.G4CheckOverlap.Resolution = cms.int32(10000)
+process.g4SimHits.G4CheckOverlap.Depth      = cms.int32(-1)
+# tells if NodeName is G4Region or G4PhysicalVolume
+process.g4SimHits.G4CheckOverlap.RegionFlag = cms.bool(False)
+# list of names
+process.g4SimHits.G4CheckOverlap.NodeNames  = cms.vstring('OCMS')
```
Comment (civanch) [inline SimG4Core/PrintGeomInfo/test/python/g4OverlapCheck_dd4hep_cfg.py:33]: @bsunanda , I suspect, that the name of the world "OCMS_1"

Comment (bsunanda) [reply SimG4Core/PrintGeomInfo/test/python/g4OverlapCheck_dd4hep_cfg.py]: Thanks - corrected it to cms:OCMS_1

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by civanch** [comment]
+1

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by civanch** [comment]
There are too many differences in the DD4Hep WF but it cannot be connected with this PR, because this PR include of update of tests which are not executed here.

**Comment by civanch** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Pull request merged**